src/main/java/skeltal/Parser.java

```diff
@@ -0,0 +1,67 @@
+package skeltal;
+
+/**
+ * A Parser class that handles the parsing of userInputs, to determine Skeltal's response.
+ */
+public class Parser {
+
+    /**
+     * This method parses the user's input and determines skeltal's response accordingly.
+     * userInput should be in the form "command arg1 arg2".
+     * @param userReply A String of the userInput.
+     */
+    public static void response(String userReply) {
+        try {
+            String[] arrReply = processReply(userReply);
+            System.out.println(Ui.line());
+            switch(arrReply[0]) {
+                case "list":
```

Comment:
I believe according to the coding standard, the case should be aligned with switch? You can configure this behaviour in Intellj here: [https://stackoverflow.com/questions/40888798/config-intellij-switch-case-style#:~:text=1%20Answer&text=Go%20to%20Settings%20%3E%20Editor%20%3E%20Code,Indent%20'case'%20branches%22.](url)